.gitignore

```diff
@@ -33,3 +33,10 @@ logs/
 
 # Conventional location for Python virtual environments.
 .venv/*
+
+# Swift compiler's cache when building Apple stats monitoring library.
+core/pkg/monitor/apple/.build/*
+
+# Build outputs that our tooling stores in the source tree.
```

Comment:
i guess in a follow up we will clean this up (or at least maybe have an option in wini?)